components/picker/index.css

```diff
@@ -10,117 +10,185 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../commons/basebutton.css";
-@import "../vars/css/components/spectrum-picker.css";
-
+@import "../commons/basebutton-coretokens.css";
 
 .spectrum-Picker {
-  @inherit: %spectrum-BaseButton;
-
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
-  /* Truncate if menu options make us too wide */
-  max-inline-size: 100%;
-  inline-size: var(--spectrum-picker-texticon-width);
-  min-inline-size: var(--spectrum-picker-texticon-min-width);
-  block-size: var(--spectrum-picker-texticon-height);
-
-  margin: 0;
-  padding-block: 0;
-
-  /* Start with text-only padding */
-  padding-inline: var(--spectrum-picker-textonly-padding-left-adjusted) var(--spectrum-picker-textonly-padding-right-adjusted);
-
-  border-width: var(--spectrum-picker-texticon-border-size);
-  border-style: solid;
-  border-radius: var(--spectrum-picker-texticon-border-radius);
-
-  transition: background-color var(--spectrum-global-animation-duration-100),
-    box-shadow var(--spectrum-global-animation-duration-100),
-    border-color var(--spectrum-global-animation-duration-100);
-
-  &:disabled,
-  &.is-disabled {
-    border-width: var(--spectrum-picker-texticon-disabled-border-size);
-    cursor: default;
+  /* Placeholder tokens */
+  .spectrum--medium {
```

Comment:
These were the existing offsets for aligning popover with picker.
<img width="330" alt="Screen Shot 2022-10-17 at 11 37 56 PM" src="https://user-images.githubusercontent.com/16216104/196206763-9e2a4cf9-e5a7-4d08-8675-0dfd4506975e.png">